### ISSUE TITLE
no module named cntk._cntk_py on Python 3.5.2 (window 7 x64)

### ISSUE THREAD

**Issue by alhabshi3k** [Microsoft/CNTK#1361]
I am running cntk-2.0 (beta9.0) on Python 3.5.2 Windows 7.
The installation process using PIP is Ok, where i am using WinPython distribution. 

But , i tried to run tutorials Python Jupyter notebooks, specifically the first tutorial "CNTK_101_LogisticRegression.ipynb" and its first cell , then the error come out.

'
---------------------------------------------------------------------------
ImportError Traceback (most recent call last)
I:\WinPython-64bit-3.5\python-3.5.2.amd64\lib\site-packages\cntk\cntk_py.py in swig_import_helper()
17 try:
---> 18 return importlib.import_module(mname)
19 except ImportError:

I:\WinPython-64bit-3.5\python-3.5.2.amd64\lib\importlib_init_.py in import_module(name, package)
125 level += 1
--> 126 return _bootstrap._gcd_import(name[level:], package, level)
127

I:\WinPython-64bit-3.5\python-3.5.2.amd64\lib\importlib_bootstrap.py in _gcd_import(name, package, level)

I:\WinPython-64bit-3.5\python-3.5.2.amd64\lib\importlib_bootstrap.py in find_and_load(name, import)

I:\WinPython-64bit-3.5\python-3.5.2.amd64\lib\importlib_bootstrap.py in find_and_load_unlocked(name, import)

ImportError: No module named 'cntk._cntk_py'

During handling of the above exception, another exception occurred:

ImportError Traceback (most recent call last)
in ()
5 import os
6 # from cntk import Trainer, learning_rate_schedule, UnitType
----> 7 from cntk.learner import sgd
8 from cntk.ops import *
9 # Select the right target device when this notebook is being tested:

I:\WinPython-64bit-3.5\python-3.5.2.amd64\lib\site-packages\cntk_init_.py in ()
9 import numpy as np
10
---> 11 from .core import *
12 from . import ops
13 from . import cntk_py

I:\WinPython-64bit-3.5\python-3.5.2.amd64\lib\site-packages\cntk\core.py in ()
8 from scipy import sparse
9
---> 10 from . import cntk_py
11 from .device import use_default_device, cpu
12 from .utils.swig_helper import typemap

I:\WinPython-64bit-3.5\python-3.5.2.amd64\lib\site-packages\cntk\cntk_py.py in ()
19 except ImportError:
20 return importlib.import_module('_cntk_py')
---> 21 _cntk_py = swig_import_helper()
22 del swig_import_helper
23 elif _swig_python_version_info >= (2, 6, 0):

I:\WinPython-64bit-3.5\python-3.5.2.amd64\lib\site-packages\cntk\cntk_py.py in swig_import_helper()
18 return importlib.import_module(mname)
19 except ImportError:
---> 20 return importlib.import_module('_cntk_py')
21 _cntk_py = swig_import_helper()
22 del swig_import_helper

I:\WinPython-64bit-3.5\python-3.5.2.amd64\lib\importlib_init_.py in import_module(name, package)
124 break
125 level += 1
--> 126 return _bootstrap._gcd_import(name[level:], package, level)
127
128

ImportError: DLL load failed: The specified module could not be found.'

I found similar problem issued , but it was assumed it was for Arch Linux [https://github.com/Microsoft/CNTK/issues/1194]

**Comment by stonebig** [comment]
on current WinPython-64bit-3.5.3.0Qt5rc.exe:
- download the CNTK-2-0-beta9-0-Windows-64bit-CPU-Only.zip
- extract and install MSMpiSetup.exe (Microsoft message passing system)
- extract and install wheel cntk-2.0.beta9.0-cp35-cp35m-win_amd64.whl
- extract the "Tutorial" directory and install in "notebook" 

... seems to work on Windwos 10, at least.

**Comment by stonebig** [comment]
````
'not found (required by /usr/lib/openmpi/libmpi_cxx.so.1)
````

==> you forgot to install the MSMpiSetup.exe (Microsoft message passing system)

**Comment by alhabshi3k** [comment]
What about CNTK supporting GPU and 1bit-SGD ? 
- I have Cuda 8 with CuDNN, i want to utilize my GPU .  is it still under development ?

**Comment by paddyhoran** [comment]
I followed the installation instructions as closely as I could and got this same error (no module named cntk._cntk_py) while installing on Window 7 (manual install).  I just wanted to add a note that I had to manually add the /cntk/cntk folder to my PATH environment variable and that fixed the issue.

The documentation states the following in step 4:

> This script will also add your CNTK installation to your current PATH environment.  

It did not seem to do this for me.  However, the documentation tells you to:

> cd C:\local\Anaconda3-4.1.1-Windows-x86_64\scripts

Which I do not as I use PyCharm.  Anyway, just wanted to leave a note for others.
